tests/queries/0_stateless/02980_s3_plain_DROP_TABLE_MergeTree.reference

```diff
@@ -3,28 +3,28 @@ data after ATTACH	1
 Files before DETACH TABLE
 all_1_1_0
 
-backups/ordinary_default/data/ordinary_default/data/all_1_1_0:
-primary.cidx
-serialization.json
-metadata_version.txt
-default_compression_codec.txt
+/backups/ordinary_default/data/ordinary_default/data/all_1_1_0:
```

Comment:
Sorted in the alphabetical order (seems logical, because otherwise the order is not specified). And add / here (I checked, UNIX utility prints a relative path if a relative path was given as input and an absolute path otherwise). $path is an absolute here, I checked